docs/stdlib/math.rst

```diff
@@ -124,7 +123,7 @@ Math
 
     :index: average avg
 
-    Return the arithmetic mean of the input set.
+    Returns the arithmetic average/mean value of ``x``:
```

Comment:
```suggestion
    Returns the arithmetic mean (i.e., average) of the input set.
```

I like the idea of adding "average" as a synonym for "arithmetic mean," but I've rephrased it to be clear that's what we're doing. Also reverted to "input set" since that is what this function operates on.